README.md

```diff
@@ -1,39 +1,28 @@
 # rviz_satellite
 
-Plugin for rviz for displaying satellite maps loaded from the internet.
+Rviz plugin for displaying satellite maps at the position of a `sensor_msgs/msg/NavSatFix` message.
 
-![Alt text](.screenshot.png?raw=true "Example Image")
+## Usage
 
-In the near future this plugin will not add support for ROS 2.
-For an unofficial ROS 2 fork see [blacksoul000/rviz_satellite](https://github.com/blacksoul000/rviz_satellite/tree/dashing).
+Add an instance of `AerialMap` to your Rviz config and point it to a `sensor_msgs/msg/NavSatFix` topic.
 
-In order to use rviz_satellite, add this package to your catkin workspace.
+Map tiles will be cached to `$HOME/.cache/rviz_satellite`.
+At present the cache does not expire automatically - you should delete the files in the folder if you want the images to be reloaded.
 
-## Demo
+Currently, the [OpenStreetMap](http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames) convention for tile URLs is supported.
+This e.g. implies that only raster tiles (no vector tiles) are supported.
 
-The package contains a launch file for demonstration purposes.
-Use it to verify your installation and to get started:
+To try a demo, run the following commands to make Rviz circle around a geo point you define.
 
+```bash
+rviz2 -d $(ros2 pkg prefix --share rviz_satellite)/launch/demo.rviz
```

Comment:
These files are not related to any launch.

I suggest to put the rviz config and the publish node in a `demo` folder instead of `launch`

Reply:
I went ahead and moved this to a `demo` dir and added the testing script to be installed so it can be ran via `ros2 run`.